backend/test/identityAggregator.js

```diff
@@ -17,29 +18,57 @@ const {
 // chai.use(solidity);
 
 
-describe("IdentityAggregator", function () {
+describe.only("IdentityAggregator", function () {
```

Comment:
Did you mean to push this with describe.only? I'd imagine that may cause unintended difficulties later when only this test runs.

Otherwise, all looks great

Reply:
cool, i see it's fixed

Reply:
Accidentally left this in before pushing